src/index.html

```diff
@@ -11,7 +11,192 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" onsubmit="onSubmit()"
+    method="POST">
+      <div class="information">
+        <fieldset>
+          <legend>Personal information</legend>
+          <label for="surname" class="form-field">
+        Surname:
+            <input
+        type="text"
+        name="surname"
+        id="surname"
+        autocomplete="off"
+        required
+        >
+          </label>
+
+          <label for="name" class="form-field">
+              Name:
+            <input
+              type="text"
+              name="name"
+              id="name"
+              autocomplete="off"
+              required
+              >
```

Comment:
```suggestion
            <input
              type="text"
              name="name"
              id="name"
              autocomplete="off"
              required
            >
```
Closing bracket should be on the same level of indentation as the opening one